packages/babel-preset-cozy-app/README.md

```diff
@@ -22,7 +22,8 @@
 
 A shareable configuration for Cozy Applications or Scripts.
 
-This package is a Babel preset already used by [`create-cozy-app`](https://github.com/CPatchane/create-cozy-app).
+This package is a Babel preset used by all our Applications and libs at Cozy. It comes 
+with a few presets and options but also remove PropTypes in the production build.
```

Comment:
"few presets and options" is vague, I think starting a list of features would be good.

Features:

- Removing of PropTypes in production build
- Optionally converts ES6 imports to ES5 requires (deactivated for libs, important for down the stream tree shaking)
- and more...

With a list, it is easier to add features as we see them in the code :)

Reply:
I was too lazy to do it. Thanks!